core/libp2p/crypto/key.hpp

```diff
@@ -16,7 +16,7 @@ namespace libp2p::crypto {
     /**
      * Supported types of all keys
      */
-    enum class Type { UNSPECIFIED, RSA1024, RSA2048, RSA4096, ED25519 };
+    enum class Type { UNSPECIFIED, RSA1024, RSA2048, RSA4096, ED25519, SECP256K1 };
     // TODO(yuraz): add support for Secp256k1 like in js version (added to
     // PRE-103)
```

Comment:
Haven't you just done it? If yes, please remove the TODO

Reply:
already removed, please include all commits.